specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json

```diff
@@ -481,6 +481,141 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/locations/{location}/checkNameAvailability": {
+      "post": {
+        "x-ms-examples": {
+          "Get details of a server": { "$ref": "./examples/checkNameAvailability.json" }
+        },
+        "tags": [
+          "Servers"
+        ],
+        "operationId": "Servers_CheckNameAvailability",
+        "description": "Check the name availability in the target location.",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The region name which the operation will lookup into."
+          },
+          {
+            "name": "serverParameters",
+            "in": "body",
+            "required": true,
+            "schema":
+            { 
+              "$ref": "#/definitions/CheckServerNameAvailabilityBody"
```

Comment:
A better name is `CheckServerNameAvailabilityParameters`

Reply:
Thanks for the review, I updated them, please take a look once the build is completed.